### PR TITLE
Default API and MongoDB endpoints to localhost

### PR DESCRIPTION
**Pull request by erdnaxe**
First of all, thank you for sharing this awesome code!

This pull request proposes to default all API and MongoDB endpoints to `localhost` when the respective environment variable is absent. This seems to be a sanier default than hardcoding containers hostname in the code.
It also renames `REACT_APP_FLAG_REGEX` to `FLAG_REGEX`, which might have been a bug.

Best regards,

### CODE REVIEW

File: services/configurations.py [hunk 1]
```diff
@@ -38,6 +38,6 @@
 
 def containsFlag(text):
```
Comment (RickdeJager) [inline services/configurations.py:39]: This is actually dead code, left over from flower. The `containsFlag` check is now done in the assembler instead.  
  
I've already removed it in my private fork for next ECSC, but forgot to do it here. You can remove this entire function.

Comment (erdnaxe) [reply services/configurations.py]: Removed, thanks!

**Comment by erdnaxe** [comment]
image

**Comment by RickdeJager** [comment]
Thanks!

**Pull request merged**